include/CLI/LeapFormatter.hpp

```diff
@@ -150,34 +97,25 @@ class LeapFormatter : public FormatterBase {
                case AppFormatMode::SubCompact:
                   out << tree_symbol << make_expanded(new_com, mode);
                   break;
+               default:
+                  break;
```

Comment:
Do you want to log an error, in case a real bug happens?

Reply:
Done! It will throw now if that happens (but should never happen). Thank you!